src/content/docs/opentelemetry/nginx/nginx-otel-host.mdx

```diff
@@ -0,0 +1,409 @@
+   Environment="NEWRELIC_LICENSE_KEY=YOUR_LICENSE_KEY"
+   EOF
+   ```
+3. Reload systemd and restart the collector:
+   ```bash
+   sudo systemctl daemon-reload
+   sudo systemctl restart otelcol-contrib.service
+   ```
+
+## Step 5: Forward NGINX logs [#forward-logs]
+
+Extend your collector configuration to include access and error logs if you want log events alongside metrics.
+
+### Configure NGINX log format
+
+Before forwarding logs, configure NGINX to use a structured log format. Refer to the [NGINX log module documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for guidance on configuring access and error logs.
+
+### Configure the OpenTelemetry Collector for log forwarding
+
+1. Note the full paths to your NGINX access and error log files. Defaults are usually `/var/log/nginx/access.log` and `/var/log/nginx/error.log`.
+
+2. Update `/etc/otelcol-contrib/config.yaml` to add a `filelog` receiver and log pipeline:
+
+   ```yaml
+   receivers:
+     nginx:
+       # existing stub status receiver configuration
+     filelog/nginx_access_logs:
+       include:
+         - /var/log/nginx/access.log
+     filelog/nginx_error_logs:
+       include:
+         - /var/log/nginx/error.log
+
+   processors:
+     resourcedetection:
+       # existing settings
+     resource:
+       # existing settings
+     batch:
+       # existing settings
+     transform/nginx_metrics:
+       # existing settings
+     transform/nginx_access_logs:
+       log_statements:
+         - context: resource
+           statements:
+             - set(attributes["nginx.display.name"], Concat(["server", attributes["nginx.deployment.name"]], ":"))
+             - set(attributes["logtype"], "nginx")
+     transform/nginx_error_logs:
+       log_statements:
+         - context: resource
+           statements:
+             - set(attributes["nginx.display.name"], Concat(["server", attributes["nginx.deployment.name"]], ":"))
+             - set(attributes["logtype"], "nginx-error")
+
+   exporters:
+     # existing exporter setup
+
+   service:
+     pipelines:
+       metrics:
+         receivers: [nginx]
+         processors: [resourcedetection, resource, batch, transform/nginx_metrics]
+         exporters: [otlphttp/newrelic]
+       logs/nginx-access:
+         receivers: [filelog/nginx_access_logs]
+         processors: [batch, resource/nginx, transform/nginx_access_logs]
+         exporters: [otlphttp/newrelic]
+       logs/nginx-error:
+         receivers: [filelog/nginx_error_logs]
+         processors: [batch, resource/nginx, transform/nginx_error_logs]
+         exporters: [otlphttp/newrelic]
+   ```
+
+3. Grant the `otelcol-contrib` user read access to the log files:
+
+   ```bash
+   sudo usermod -a -G adm otelcol-contrib
+   sudo chmod 644 /var/log/nginx/access.log
+   sudo chmod 644 /var/log/nginx/error.log
+   ```
+
+4. Restart the collector to apply the changes:
+
+   ```bash
+   sudo systemctl restart otelcol-contrib
```

Comment:
Why is this needed again? Since the logs are a mandatory step can we have everything included in a single step? May I know why was this split.